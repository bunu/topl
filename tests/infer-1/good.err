Capturing in javac mode...
Found 2 source files to analyze in /home/rg/research/code/topl/tests/infer-1/infer-out
Starting analysis...

legend:
  "F" analyzing a file
  "." analyzing a procedure

FF............................................................................................................
Found 2 issues

Main.java:13: warning: PRECONDITION_NOT_MET
  in call to `i4(...)` at line 13.
  11.       Bad.s4(null, null, null, null);
  12.       Bad x = new Bad(null, null, null);
  13. >     x.i4(null, null, null, null);
  14.     }
  15.   }

Main.java:12: warning: PRECONDITION_NOT_MET
  in call to `Bad(...)` at line 12.
  10.     public static void main(String[] args) {
  11.       Bad.s4(null, null, null, null);
  12. >     Bad x = new Bad(null, null, null);
  13.       x.i4(null, null, null, null);
  14.     }


