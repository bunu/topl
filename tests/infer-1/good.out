
Summary of the reports

  PRECONDITION_NOT_MET: 2
